code/modules/mob/living/simple_animal/abnormality/waw/red_riding_mercenary.dm

```diff
@@ -43,7 +43,7 @@ It has now been over four months. Now we get her for real. -Coxswain
 	faction = list("redhood") // I'LL FUCKIN FIGHT YOU TOO, MATE
 	can_breach = TRUE
 	start_qliphoth = 3
-
+	density = FALSE //Prevents red from getting stuck unable to reach her target
```

Comment:
You dont have to do this but Mountain of Smiling Bodies has something to bypass being stuck on people when patrolling. 
```
/mob/living/simple_animal/hostile/abnormality/mountain/CanPassThrough(atom/blocker, turf/target, blocker_opinion)
	if(phase <= 1 && isliving(blocker)) // Phase 1 MoSB cannot be blocked by mobs when it's moving towards corpses
		return TRUE
	return ..()
```